lib/pages/forms_pages/components/whatsapp_camera.dart/camera_whatsapp.dart

```diff
@@ -196,6 +198,28 @@ class _WhatsappCameraState extends State<WhatsappCamera>
               },
             ),
           ),
+          Visibility(
+            visible: widget.isAdultReport,
+            child:Align(
+              alignment: Alignment.topCenter,
+              child: Padding(
+                padding: const EdgeInsets.only(top: 70),
+                child: Container(
+                  width: 0.5 * MediaQuery.of(context).size.width,
+                  decoration: BoxDecoration(
+                    color: Colors.orange,
```

Comment:
Could you try with `Colors.orange` with a degree of shade (`Color.orange.shade300` or `Color.orange.shade200` maybe??)? If you think the UI is improved go ahead

Reply:
it looks lighter, let's go ahead